internal/notifier/webex.go

```diff
@@ -23,23 +23,70 @@ import (
 	"strings"
 
 	"github.com/fluxcd/pkg/runtime/events"
+	"github.com/hashicorp/go-retryablehttp"
 )
 
+// Example of provider manifest for webex:
```

Comment:
Please move the example to the API docs right under this section https://github.com/fluxcd/notification-controller/blob/main/docs/spec/v1beta1/provider.md#slack-app

Reply:
done